airflow/dags/download_gtfs_schedule_v2/download_feeds.py

```diff
@@ -0,0 +1,135 @@
+# ---
+# python_callable: download_all
+# provide_context: true
+# ---
+import cgi
+import logging
+import traceback
+
+import os
+import pendulum
+import re
+from airflow.models import Variable
+from airflow.utils.db import create_session
+from pydantic.networks import HttpUrl
+from pydantic.tools import parse_obj_as
+from requests import Session
+from typing import List, Dict
+from calitp.storage import get_fs
+from utils import (
+    AirtableGTFSDataExtract,
+    AirtableGTFSDataRecord,
+    AirtableGTFSDataRecordProcessingOutcome,
+    GTFSFeedExtractInfo,
+    GTFSFeedType,
+)
+
+GTFS_FEED_LIST_ERROR_THRESHOLD = 0.95
+
+
+def download_feed(
+    record: AirtableGTFSDataRecord, auth_dict: Dict
+) -> AirtableGTFSDataRecordProcessingOutcome:
+    if not record.uri:
+        raise ValueError("")
+
+    parse_obj_as(HttpUrl, record.uri)
```

Comment:
I think we do want HttpUrl as it's a bit stricter, especially for checking there's a good TLD... I could be convinced otherwise.

https://pydantic-docs.helpmanual.io/usage/types/#urls